report/11_Systems-Design.md

```diff
@@ -1,9 +1,10 @@
 ### System Design 
+
 Our version of the MiniTwit application is a web app written in C# using ASP.NET. Data is stored using Entity Framework Core (EF Core) as a database abstraction layer working with a PostgreSQL database. This stack was chosen because it has both excellent support, maintenance, and documentation. Furthermore, this means that many third-party packages are available, like SeriLog which makes interaction with ElasticSearch painless and Prometheus-net for plug and play communication with Prometheus.
 
 The application has three main C# Projects.
 
-- MiniTwit.Web.App, which contains the web app that users interact with.
+- MiniTwit.Web.App, which contains the web app and api that users interact with.
```

Comment:
I think we are inconsistent about whether it's api or API, we should probably make a decision about it

Reply:
Use API